### PR TITLE
nix/nixos install notes

### PR DESCRIPTION
**Pull request by 06kellyjac**
# Changes

Added notes for installing through nix

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- N/A - Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- N/A - Run the code checkers with `make check`
- N/A - Regenerate the manpages, docs and go formatting with `make generated`
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,14 @@ Download the latest binary executable for your operating system:
   dpkg -i tektoncd-cli-0.16.0_Linux-64bit.deb
   ```
 
+### NixOS/Nix
+
+You can install `tektoncd-cli` from [nixpkgs](https://github.com/NixOS/nixpkgs) on any system that supports the `nix` package manager.
```
Comment (vdemeester) [inline README.md:139]: Oh waw it got in 😅 I kept updating my own derivation ([here](https://git.sr.ht/~vdemeester/home/tree/master/item/nix/packages/tkn/default.nix)), I guess I should update nixpkgs from now on 😛 (I wash shy to propose it at first, and then time did bite me)

**Review by vdemeester — APPROVED** [review]
Awesome @06kellyjac @mstrangfeld 🕺🏼 
/meow

File: README.md (hunk 1 above)
Comment (06kellyjac) [reply README.md:139]: You're welcome to add yourself as a maintainer :)

**Comment by 06kellyjac** [comment]
Anything else required for the PR?
Are those tests failing due to my changes?

**Comment by chmouel** [comment]
@tektoncd/cli-maintainers the tests are getting quite flaky, we need to talk abotu disabling or fixing some of them,

**Pull request merged**